src/qasm-simulator-cpp/Makefile

```diff
@@ -1,12 +1,195 @@
-.SUFFIXES:.cc .o .c
```

Comment:
Can you check if the changes to this file were on purpose? It seems it might have slipped through.

Reply:
Good catch! Thanks, I'll correct.